src/ensembl/src/content/home/Home.tsx

```diff
@@ -15,25 +15,160 @@
+                      <span>Genome browser</span>
+                      <BrowserIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Look at genes & transcripts in their genomic context
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.selectionContainer}>
+                <div className={styles.buttonsContainer}>
+                  <Link to="/entity-viewer">
+                    <div className={styles.button}>
+                      <span>Entity viewer</span>
+                      <EntityViewerIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Get gene & transcript information
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {isEnvironment([Environment.DEVELOPMENT, Environment.INTERNAL]) ? (
+            <div className={styles.homeTopRight}>
+              <Link to="/help">{helpButton}</Link>
+            </div>
+          ) : (
+            <div
+              className={classNames(
+                styles.homeTopRight,
+                styles.homeTopRightDisabled
+              )}
+            >
+              {helpButton}
+            </div>
+          )}
+        </div>
+
+        <div className={styles.speciesContainer}>
+          <span className={styles.platform}></span>
+          <img src={speciesStripUrl} className={styles.speciesStrip} />
+        </div>
+      </div>
+      <div className={styles.footer}>
+        <div className={styles.footerLeft}>
+          <Logotype className={styles.logotype} />
+          <img src={ebiLogoUrl} className={styles.ebiLogo} />
+          <div className={styles.text}>
+            Wellcome Genome Campus, Hinxton, Cambridgeshire CB10 1SD, UK
+          </div>
+        </div>
+
+        <div className={styles.footerRightIcons}>
+          <a
+            href="//www.ensembl.info"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className={styles.link}>Ensembl blog</div>
+          </a>
+          <a
+            href="//www.ensembl.info"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src={blogIconUrl} className={styles.mediaIcons} />
+          </a>
+          <a
+            href="//www.facebook.com/Ensembl.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img src={facebookIconUrl} className={styles.mediaIcons} />
+          </a>
+          <a
+            href="//www.twitter.com/ensembl"
```

Comment:
twitter account doesn't have a `www` in the url:

![image](https://user-images.githubusercontent.com/6834224/123655678-1904c800-d827-11eb-8635-72cab8069479.png)

Reply:
I have removed the `www` but the link does work with `www`.

Reply:
It does, but twitter redirects it to no www:

![image](https://user-images.githubusercontent.com/6834224/123939281-e4605000-d98f-11eb-888d-b63b42b496ce.png)